android/src/main/res/layout/player_controls.xml

```diff
@@ -27,16 +27,16 @@
             app:layout_constraintStart_toEndOf="@+id/rewind_guideline"
             app:layout_constraintTop_toTopOf="parent" />
 
-        <ImageButton
+       <ImageButton
             android:id="@+id/close_btn"
             android:layout_width="@dimen/action_icon_size"
             android:layout_height="@dimen/action_icon_size"
             android:layout_marginEnd="@dimen/close_button_margin_start"
             android:background="@android:color/transparent"
             android:src="@drawable/ic_close"
-            app:layout_constraintBottom_toBottomOf="@id/title"
+            app:layout_constraintTop_toTopOf="parent"
             app:layout_constraintStart_toStartOf="@id/title"
-            app:layout_constraintTop_toTopOf="@id/title" />
+            app:layout_constraintLeft_toRightOf="parent" />
```

Comment:
@Kanchan-Salotra-brightcove we already have a constraintStart set. That is used instead of constraintLeft. Also, setting constraintLeft_toRightOf doesn't make sense when the target is parent. If your goal is to keep the same positioning as the close icon while hiding the title, I recommend using INVISIBLE instead of GONE for the title visibility and keeping the existing constraints for the close button